pages/common/dmesg.md

```diff
@@ -0,0 +1,11 @@
+# dmesg
+
+> Write the kernel messages in Linux and other Unix-like operating systems to standard output.
```

Comment:
Since we would be splitting the page into per-os level, we can remove the "Linux and other Unix-like operating systems".

So there, I suggest - 
```
Display kernel messages to standard output.
```